Pipfile

```diff
@@ -13,15 +13,11 @@ legistar = {git = "https://github.com/opencivicdata/python-legistar-scraper"}
 scrapy-wayback-middleware = "*"
 city-scrapers-core = {extras = ["azure"],version = "*"}
 pypdf2 = "*"
-pywin32 = {version = "*",sys_platform = "== 'win32'"}
 
 [dev-packages]
```

Comment:
The Pipfile shouldn't be changed, so `pywin32`, `pathlib2` and the scripts should be in place still and `pylint` shouldn't be added